specs/shell/array_create.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: Makes a new, empty array
```

Comment:
nit: Could you make sure the tense is consistent for all these workflows?

I think declarative tense (e.g. "Make a new, empty, array") would be slightly clearer here

Reply:
Fixed!